logging/src/commonMain/kotlin/log.kt

```diff
@@ -0,0 +1,267 @@
+package com.splendo.mpp.log
+
+import ru.pocketbyte.hydra.log.HydraLog
+
+/**
+ * Initializes HydraLog with given logger.
+ *
+ * Be aware that once initialized HydraLog cannot be initialised with new loggers. You will get first logger you have used for initialisation.
+ *
+ * @param logger - logged to be used for log output
+ * @return first logger used for initialisation.
+ */
+fun initLogger(logger: Logger): Logger? {
```

Comment:
Still no reason for the return here to be optional. We should either initialize a logger, return the existing one (in some cases after initialization fails), or just throw an error if everything goes wrong.